data/migrations/20211101133234_application_tickets.js

```diff
@@ -14,6 +14,8 @@ exports.up = function (knex) {
       table
         .string('profile_id')
         .notNullable()
+        .unsigned()
+        .notNullable()
```

Comment:
This looks like a mistake; I don't think you need two `.notNullable()` calls, and since the ID is a string, `.unsigned()` seems out of place, too (it's for numbers, not strings, if I recall correctly -- it basically says "don't allow negatives").